SX1276/SX1276_LoRaRadio.h

```diff
@@ -45,6 +44,14 @@ SPDX-License-Identifier: BSD-3-Clause
 #define MAX_DATA_BUFFER_SIZE_SX1276                        255
 #endif
 
+#if DEVICE_LPTICKER
```

Comment:
Why here? Should be defined in base class

Reply:
Because Base class doesn't define any timers. Timers are local to implementation. Base class just enforces an API template.